nodejs/connection/TypeDBDriverImpl.ts

```diff
@@ -57,15 +59,16 @@ export class TypeDBDriverImpl implements TypeDBDriver {
         this._initAddresses = addresses;
         this._credential = credential;
 
-        this._isOpen = false
+        this._isOpen = false;
+        this._isEnterprise = credential == null;
```

Comment:
naive quesiton but isnt credential == null actually a condition to _don't_ be EnterpriseDriver?

Reply:
You're absolute right! Thank you!